libmisc/sub.c

```diff
@@ -27,6 +29,18 @@
  */
 void subsystem (const struct passwd *pw)
 {
+	static int depth;
```

Comment:
Hm, but depth will only be initialized to 0 if it's BSS.  Does being 'static' but defined inside a function qualify?  I did not think it did.

Reply:
I have updated the commit to initialize depth with 0. Even though I think that it will always be 0 initialized, it matches other situations in which static variables are used.

Reply:
Hm, but the patch doesn't show it being initialized.

Did that change get lost?

Reply:
I'm sorry for that. The rework of goto to a function call was easier for me with a reset of the branch. And unfortunately I have reset this change as well. It is in the branch again. Thank you for carefully reviewing everything!